typhos/positioner.py

```diff
@@ -314,7 +314,10 @@ def _link_limits_by_limits_attr(self):
         except Exception:
             ...
         else:
-            if low_limit < high_limit:
+            if low_limit is None or high_limit is None:
+                # TODO: a better fix? need to wait for connection?
+                ...
```

Comment:
It might be possible to have one more fallback that is subscribing to the setpoint PV's metadata callback- but other than that, there's not that much more we can do if we want to rely on an attribute instead of a signal.

Reply:
I think you're right. What gave me pause mostly was the fact that we hide the limit widgets in this scenario - perhaps we just say "well, show the widgets at least so they can be set"

... In typing the above, though, I think there's no method for setting `.limits` with these widgets (in the fallback scenario you accurately described [in this comment](https://github.com/pcdshub/typhos/pull/567#issuecomment-1672144529)) - right? Or am I missing it?

Reply:
Right, these widgets are read-only. I think they are just the little text tags that go under the limit switch indicators.